pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [build-system]
-requires = ["setuptools", "wheel", "cmake"]
+requires = ["setuptools", "ninja; sys_platform != 'win32'", "wheel", "cmake"]
```

Comment:
As of now it should work with versions that are so common, that I don't know the minimum we should require.

Is setting minimum versions here something we should do as we are removing setup.py at some point in the future (or using more recent features of `setuptools` and co)?

Reply:
at least for cmake it would probably make sense to specify `cmake>=3.14`. For the others we do not have particular requirements at the moment.

In the future, we will probably move to PEP 621 style `pyproject.toml` for describing project metadata. this will then require at least `setuptools>=61` (probably higher).